pkg/apis/logging/v1/output.go

```diff
@@ -0,0 +1,72 @@
+package v1
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1/outputs"
+)
+
+const OutputTypeDefault = "default"
```

Comment:
is there a reason we have "default" as a default type rather than it being the same as `OutputTypeElasticsearch` ?

Reply:
Yep - someday elasticsearch may not be the default. OutputTypeElasticSearch means "I want to forward to an elastic instance (and I'll provide the URL and other necessary config)" whereas "default" means "I want to forward to the openshift logging default store, whatever that is, however it's configured".